de.metas.document.archive/de.metas.document.archive.base/src/main/java/de/metas/document/archive/api/impl/DocOutboundDAO.java

```diff
@@ -59,6 +59,20 @@ public I_C_Doc_Outbound_Log retrieveLog(final I_AD_Archive archive)
 
 		return docExchange;
 	}
+	
+	@Override
+	public I_C_Doc_Outbound_Log retrieveLog(Properties ctx, int bpartnerId, int AD_Table_ID)
```

Comment:
Pls use the IQueryBL instead of hardcoded whereclauses, in order to achieve testability without a DB.
After that, you can move this method to AbstractDocOutboundDAO